app-modules/ai/tests/Tenant/Feature/Jobs/EmailAiThreadTest.php

```diff
@@ -34,4 +34,39 @@
 </COPYRIGHT>
 */
 
-todo('can send a notification containing a thread transcript');
+use AdvisingApp\Ai\Enums\AiApplication;
+use AdvisingApp\Ai\Enums\AiModel;
+use AdvisingApp\Ai\Jobs\EmailAiThread;
+use AdvisingApp\Ai\Models\AiAssistant;
+use AdvisingApp\Ai\Models\AiMessage;
+use AdvisingApp\Ai\Models\AiThread;
+use AdvisingApp\Ai\Notifications\AssistantTranscriptNotification;
+use AdvisingApp\Authorization\Enums\LicenseType;
+use App\Models\User;
+use Illuminate\Support\Facades\Notification;
+
+use function Pest\Laravel\withoutMiddleware;
+
+it('can send a notification containing a thread transcript', function () {
+    Notification::fake();
+    withoutMiddleware();
+
+    $sender = User::factory()->licensed(LicenseType::cases())->create();
+    $recipient = User::factory()->licensed(LicenseType::cases())->create();
+
+    $assistant = AiAssistant::factory()->create([
+        'application' => AiApplication::Test,
+        'is_default' => true,
+        'model' => AiModel::Test,
+    ]);
+
+    $thread = AiThread::factory()
+        ->for($assistant, 'assistant')
+        ->has(AiMessage::factory()->count(3), 'messages')
+        ->for($sender, 'user')
+        ->create();
+
+    dispatch(new EmailAiThread($thread, $sender, $recipient));
```

Comment:
Rather than `dispatch` this it can be better to just new up the job class and trigger the handle method directly.

As the purpose of this test is to specifically test the functionality of this job.

Reply:
As above